spec/controllers/coronavirus/announcements_controller_spec.rb

```diff
@@ -98,20 +98,21 @@
       }
     end
 
-    subject { delete :destroy, params: announcement_params }
-
     it "redirects to the coronavirus page" do
-      expect(subject).to redirect_to(coronavirus_page_path(page.slug))
+      delete :destroy, params: announcement_params
+      expect(response).to redirect_to(coronavirus_page_path(page.slug))
     end
 
     it "deletes the announcement" do
-      expect { subject }.to change { Coronavirus::Announcement.count }.by(-1)
+      expect { delete :destroy, params: announcement_params }
+        .to change { Coronavirus::Announcement.count }.by(-1)
     end
 
     it "doesn't delete the announcement if draft_updater fails" do
       stub_publishing_api_isnt_available
 
-      expect { subject }.to_not(change { page.reload.announcements.to_a })
+      expect { delete :destroy, params: announcement_params }
+        .to_not(change { page.reload.announcements.to_a })
```

Comment:
```suggestion
        .not_to change({ Coronavirus::Announcement.count })
```

Sorry I know the previous code isn't yours. It seems a bit strange that we test a deletion in one way and the inverse of it in a totally different way.